src/javascript/app_2/Modules/Trading/Containers/purchase.jsx

```diff
@@ -1,16 +1,17 @@
+import classNames        from 'classnames';
 import PropTypes         from 'prop-types';
 import React             from 'react';
 import { localize }      from '_common/localize';
-import { isEmptyObject } from '_common/utility';
 import { PopConfirm }    from 'App/Components/Elements/PopConfirm';
+import Tooltip           from 'App/Components/Elements/tooltip.jsx';
 import Fieldset          from 'App/Components/Form/fieldset.jsx';
 import { connect }       from 'Stores/connect';
 import ContractInfo      from '../Components/Form/Purchase/contract-info.jsx';
-import MessageBox        from '../Components/Form/Purchase/MessageBox';
 import PurchaseLock      from '../Components/Form/Purchase/PurchaseLock';
 import PurchaseButton    from '../Components/Elements/purchase-button.jsx';
```

Comment:
extra whitespace after `import { isEmptyObject }` removed.

Reply:
I don't think so, that's just how github displays it. there's no line number, it's empty.